Frontend/ui-driver/src/Flow.purs

```diff
@@ -2413,4 +2415,16 @@ getUpiApps = do
   let appsNotSupportMandate = runFn1 stringifyJSON $ map (\item -> item.appName) $ filter (\item -> not item.supportsMandate) resp
   pure $ setCleverTapUserProp "appsSupportMandate" appsSupportMandate
   pure $ setCleverTapUserProp "appsNotSupportMandate" appsNotSupportMandate
-  void $ Remote.updateDriverInfoBT (UpdateDriverInfoReq req{availableUpiApps = Just $ runFn1 stringifyJSON resp})
\ No newline at end of file
+  void $ Remote.updateDriverInfoBT (UpdateDriverInfoReq req{availableUpiApps = Just $ runFn1 stringifyJSON resp})
+
+checkDriverBlockingStatus :: GetDriverInfoResp -> FlowBT String Unit
+checkDriverBlockingStatus (GetDriverInfoResp getDriverInfoResp) = do
+  if (  (getValueToLocalStore ENABLE_BLOCKING) == "__failed" &&
+        isDateGreaterThan "2023-09-20T00:00:00" &&
+        (getDriverInfoResp.autoPayStatus == Nothing || getDriverInfoResp.autoPayStatus == Just "MANDATE_FAILURE")
```

Comment:
@Rohit4997 we can use any here instead of putting two conditions on the same variable.